src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        return Colors.values()[new Random().nextInt(Colors.values().length)].toString();
```

Comment:
you have at least 2 mistakes from checklist here, fix it

Reply:
I've changed from toString() method to name() method, this was the first mistake.
I can't find second mistake. Is it about name "Colors"(I've already fixed it)?